Gemfile.lock

```diff
@@ -209,60 +209,60 @@ GEM
       sprockets-rails
       tilt
     semantic_range (2.3.0)
-    spring (2.1.0)
+    spring (2.1.1)
     spring-watcher-listen (2.0.1)
       listen (>= 2.7, < 4.0)
       spring (>= 1.2, < 3.0)
     sprockets (4.0.2)
       concurrent-ruby (~> 1.0)
       rack (> 1, < 3)
-    sprockets-rails (3.2.1)
+    sprockets-rails (3.2.2)
```

Comment:
### [sprockets-rails](https://github.com/rails/sprockets-rails)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.2.1 → 3.2.2

[_~~change-log~~, [source-code](http://github.com/rails/sprockets-rails)_]





<details>
<summary>Commits</summary>

A change of **45** commits. See the full changes on [the compare page](https://github.com/rails/sprockets-rails/compare/v3.2.1...v3.2.2).

These are the first 10 commits:
- (a8ad698) [Remove CHANGELOG](https://github.com/rails/sprockets-rails/commit/a8ad698dcef0a9a497e12dc474f271c60529f4a7)
- (5847306) [Always use newest bundler in the CI](https://github.com/rails/sprockets-rails/commit/58473062db8ccf5f49d18a5ed16d67336d34f850)
- (760a805) [Use word “error” in error class.](https://github.com/rails/sprockets-rails/commit/760a805a9f56d3df0d4b83bd4a5a6476eb3aeb29)
- (3c7ff8a) [Extra newline in exception](https://github.com/rails/sprockets-rails/commit/3c7ff8a1aed028ae32210aabcbf6c25ed265695c)
- (857e781) [Merge pull request #414 from rails/schneems/more-error-stuff](https://github.com/rails/sprockets-rails/commit/857e781998c10e4f429699da1d47ef251844991f)
- (34ef126) [Fix typo](https://github.com/rails/sprockets-rails/commit/34ef126f34df7f3ef4efb88bdd8f00c568280455)
- (2bd49b3) [Document options for generating non-digest assets](https://github.com/rails/sprockets-rails/commit/2bd49b318a75a54115447112437587842bb5bbd1)
- (2506ef0) [Missing space in error message](https://github.com/rails/sprockets-rails/commit/2506ef0d8110e56f8453f7d5a90aa9f7da9d6a33)
- (49bf802) [Merge pull request #429 from justin808/patch-1](https://github.com/rails/sprockets-rails/commit/49bf8022c8d3e1d7348b3fe2e0931b2e448f1f58)
- (c468e41) [Point to gem fork with a friendlier name](https://github.com/rails/sprockets-rails/commit/c468e417877ae6fe82e72b0c76b8608f6e231f0a)


</details>